ncm-systemd/src/main/perl/Systemd/Systemctl.pm

```diff
@@ -365,14 +365,53 @@ sub systemctl_is_enabled
 
     my $ec = $?;
 
-    $logger->debug(2, "systemctl_command_units $proc returned ec $ec and stdout $stdout stderr $stderr");
+    $logger->debug(2, "systemctl_is_enabled $proc returned ec $ec and stdout $stdout stderr $stderr");
     # Do not test on $ec; if unit is disabled, it is-enabled also returns ec > 0
     # If there's a real issue, like unknown unit, there is no stdout.
     return $stdout || undef;
 }
 
 
 
+=pod
+
+=item systemctl_is_active
```

Comment:
this is copy/paste code from systemctl_is_enabled. can you factor out the common code in a new function and pass the `is-active` and `is-enabled` to this new private function?